vertical-pod-autoscaler/e2e/v1/admission_controller.go

```diff
@@ -47,7 +47,6 @@ var _ = AdmissionControllerE2eDescribe("Admission-controller", ginkgo.Label("FG:
 	f.NamespacePodSecurityEnforceLevel = podsecurity.LevelBaseline
 
 	ginkgo.BeforeEach(func() {
```

Comment:
can we combine this set of tests with the ones below now like you did elsewhere here?